parser/json.go

```diff
@@ -38,3 +39,32 @@ func (x JSON) Parse(b []byte) (*skogul.Container, error) {
 	err := json.Unmarshal(b, &container)
 	return &container, err
 }
+
+// RawJSON can be used when the JSON format does not conform to the final JSON format of skogul,
+// e.g. when it is used as the first step of parsing from a third party source where modifying
+// the source data structure might be hard/impossible
+type RawJSON struct{}
+
+// Parse accepts a byte slice of JSON data and marshals it into an empty skogul.Container
+func (data RawJSON) Parse(b []byte) (*skogul.Container, error) {
+
+	// The Validate() func of a container expects a timestamp to be valid.
+	// Better way to fix?
+	time := time.Now()
```

Comment:
Comment says it all really.

Reply:
In other words, though: 
* Should the parser support extracting a timestamp?
* Should a transformer support being able to override the timestamp? (Should a transformer actually be allowed to override the timestamp?)

Reply:
I'm thinking: 
1. For this example, the only valid timestamp _is_ now. It's a custom metric, it might not even have a timestamp.
2. Transformers should most certainly be able to override the timestamp. There are numerous examples where that's useful.
3. We should not do time.Now() for every received metric. We are looking at receiving many thousands of metrics per second and doing an extra systemcall for each one is wasteful. I suggest we create a skogul.Nowish() function and add a timer in in init() in common.go that updates a global timestamp, and Nowish() returns it. I think 1/10th second resolution is acceptable for the scenarios where we don't have more precise data available. But it can always be increased in the future...
4. For this PR, just using time.Now() is acceptable as long as an issue to fix it is also created.